deluge/ui/gtkui/aboutdialog.py

```diff
@@ -7,29 +7,28 @@
 # See LICENSE for more details.
 #
 
-import gtk
-import pygtk
-
+import gi
 from deluge.common import get_pixmap, get_version, open_url_in_browser
 from deluge.ui.client import client
 from deluge.ui.gtkui.common import get_deluge_icon
+from gi.repository import GdkPixbuf, Gtk
```

Comment:
This import line should be above deluge imports, isort should fix that though...